### ISSUE TITLE
lyon 0.16 does not build.

### ISSUE THREAD

**Issue by rukai** [nical/lyon#619]
I ran `cargo update` and now my project doesnt build because lyon is failing to build.
I can reproduce the issue with an otherwise empty project as well.

Might be related to the 0.17 release earlier today, https://github.com/nical/lyon/pull/617 but I dont see the 0.17 release on crates.io yet.

There are 255 errors like:
```
error[E0432]: unresolved imports `crate::geom::math`, `crate::AttributeStore`, `crate::ControlPointId`, `crate::EndpointId`, `crate::Event`, `crate::IdEvent`, `crate::PathEvent`, `crate::PositionStore`
  --> /home/rukai/.cargo/registry/src/github.com-1ecc6299db9ec823/lyon_path-0.16.2/src/lib.rs:56:9
   |
56 | pub use crate::geom::math;
   |         ^^^^^^^^^^^^^^^^^ no `math` in the root
   | 
  ::: /home/rukai/.cargo/registry/src/github.com-1ecc6299db9ec823/lyon_path-0.16.2/src/path.rs:8:13
   |
8  | use crate::{AttributeStore, ControlPointId, EndpointId, Event, IdEvent, PathEvent, PositionStore};
   |             ^^^^^^^^^^^^^^  ^^^^^^^^^^^^^^  ^^^^^^^^^^  ^^^^^  ^^^^^^^  ^^^^^^^^^  ^^^^^^^^^^^^^

error[E0412]: cannot find type `Point` in this scope
  --> /home/rukai/.cargo/registry/src/github.com-1ecc6299db9ec823/lyon_path-0.16.2/src/builder.rs:98:31
   |
98 |     fn move_to(&mut self, to: Point);
   |                               ^^^^^ not found in this scope
   |
help: consider importing one of these items
   |
77 | use crate::geom::Point;
   |
77 | use lyon_geom::Point;
   |

error[E0412]: cannot find type `Point` in this scope
   --> /home/rukai/.cargo/registry/src/github.com-1ecc6299db9ec823/lyon_path-0.16.2/src/builder.rs:101:31
    |
101 |     fn line_to(&mut self, to: Point);
    |                               ^^^^^ not found in this scope
    |
help: consider importing one of these items
    |
77  | use crate::geom::Point;
    |
77  | use lyon_geom::Point;
    |

error[E0412]: cannot find type `Point` in this scope
   --> /home/rukai/.cargo/registry/src/github.com-1ecc6299db9ec823/lyon_path-0.16.2/src/builder.rs:109:35
    |
109 |     fn current_position(&self) -> Point;
    |                                   ^^^^^ not found in this scope
    |
help: consider importing one of these items
    |
77  | use crate::geom::Point;
    |
77  | use lyon_geom::Point;
```

**Comment by BillyDM** [comment]
Yes please fix this! Some crates like the stable crates.io version of `Iced` currently depend on the `0.16` version.

**Comment by nical** [comment]
Ouch! thanks for bringing this to my attention.

**Comment by nical** [comment]
My mistake was to publish lyon_geom 0.16.3 instead of 0.17. Fixing that now, with 0.16.3 yanked things should get back to normal.

**Comment by nical** [comment]
Sorry about this. I yanked lyon_geom `0.16.3`, so if your project depends on lyon `0.16`, re-generating your Cargo.lock should get your build back to normal.

I added lyon_geom `0.17` and got the other `0.17` crates to depend on that. To avoid build issues with geom `0.16.3`/`0.17.0` colliding I yanked the `0.17.0` crates and published `0.17.1`, so if you already updated to `0.17`, the safest is to regenerate your Cargo.lock so that it picks up `0.17.1` or later.
